src/test_common.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(test)]
```

Comment:
You can also use `#![cfg(test)]` (note the esclamation mark after the hash). This means "put the whole file under the `cfg(test)` flag. Without the exclamation mark it only applies to the next line. This way you also don't need the `#[cfg(test)]` in `src/lib.rs`.

Reply:
Right. Thank you! And it seems to be fair for other test mods, so we can remove other `#[cfg(test)]` directives in `lib.rs` having those directive in appropriate source files